### PR TITLE
onAnswer and ringbackTone info

### PR DESCRIPTION
**Pull request by tbedford**
## Description

Adds some info about `ringbackTone` and `onAnswer` to the NCCO guide.

## Review

* [`ringbackTone`](https://nexmo-developer-pr-2805.herokuapp.com/voice/voice-api/guides/ncco#ringback-tone)
* [`onAnswer`](https://nexmo-developer-pr-2805.herokuapp.com/voice/voice-api/guides/ncco#on-answer)

### CODE REVIEW

**Comment by tbedford** [comment]
@peter-nexmo - would really appreciate your thoughts on this. Thanks.

File: _documentation/en/voice/voice-api/guides/ncco.md [hunk 1]
```diff
@@ -1,7 +1,10 @@
 ---
 title: Nexmo Call Control Objects
 navigation_weight: 2
-description: To tell Nexmo how to handle a phone call, you must provide Nexmo an Nexmo Call Control Objects (NCCO) when a call is placed or answered. There are various actions available, such as `talk`, `input` and `record`.
+description: >-
+  To tell Nexmo how to handle a phone call, you must provide Nexmo an Nexmo Call
+  Control Objects (NCCO) when a call is placed or answered. There are various
```
Comment (peter-nexmo) [inline _documentation/en/voice/voice-api/guides/ncco.md:6]: an NCCO (Nexmo Call Control Object)

File: _documentation/en/voice/voice-api/guides/ncco.md [hunk 2]
```diff
@@ -57,26 +62,116 @@ For your NCCO to execute correctly, the JSON syntax must be valid. It is better
...
 
 You can use these parameters to customize the NCCO you return to Nexmo. The following code examples show how to provide the NCCO that controls your call or conversation:
 
 ```tabbed_examples
 source: '_examples/voice/ncco/creating-a-custom-call-or-conversation-for-each-user'
 ```
+
+⚓ ringback-tone
+
+## Connecting to endpoints that don't generate a calling tone
```
Comment (peter-nexmo) [inline _documentation/en/voice/voice-api/guides/ncco.md:91]: calling > ring back tone

File: _documentation/en/voice/voice-api/guides/ncco.md [hunk 3]
```diff
@@ -57,26 +62,116 @@ For your NCCO to execute correctly, the JSON syntax must be valid. It is better
...
+
+Sometimes you will connect to an endpoint that does not have a call tone provided by the carrier. If calling a phone endpoint, you will usually hear a dial tone provided by the carrier, but if one is not provided for that endpoint the caller will experience silence and may be unsure if they are being connected or not. This may result in them hanging up the call. To get around this potential problem, you can use `ringbackTone` in your NCCO to provide a calling tone. For example:
```
Comment (peter-nexmo) [inline _documentation/en/voice/voice-api/guides/ncco.md:93]: dial tone > ring back tone

**Comment by tbedford** [comment]
@niati - could you ensure I have satisfied Peter's suggestions please? Many thanks!

**Pull request merged**